Miguel tuñón 8-929-2198  Henry Alvarado 8-939-1856/lab7arreglos.c

```diff
@@ -0,0 +1,19 @@
+#include<stdio.h>
+void main()
+{
+	int c,i,n[100],j[100];
+	
+	printf("Ingrese la cantidad de numeros\n");
+	scanf("%d",&c);
+	
+	for(i=0;i<c;i++)
+	{

```

Comment:
Se debia hacer un ciclo for para la impresion, y para la asignacion. Como lo hiciste todo, en el mismo for.  se ven todos los numeros corriedos